### PR TITLE
[DAT-725] - Assign the ExtraCreditsPromotion in the BillingCredits

### PR DESCRIPTION
**Pull request by jhoffman-ms**
Assign the ExtraCreditsPromotion in the BillingCredits

**Task:** [DAT-725](https://makingsense.atlassian.net/jira/software/projects/DAT/boards/62?selectedIssue=DAT-725)

### CODE REVIEW

File: Doppler.BillingUser/Infrastructure/BillingRepository.cs [hunk 1]
```diff
@@ -669,7 +670,7 @@ public async Task<int> CreateAccountingEntriesAsync(AgreementInformation agreeme
                 @creditsQty = buyCreditAgreement.BillingCredit.CreditsQty,
                 @activationDate = now,
                 @extraEmailFee = buyCreditAgreement.BillingCredit.ExtraEmailFee,
-                @totalCreditsQty = buyCreditAgreement.BillingCredit.CreditsQty,
+                @totalCreditsQty = buyCreditAgreement.BillingCredit.CreditsQty + buyCreditAgreement.BillingCredit.ExtraCreditsPromotion,
```
Comment (nreal) [inline Doppler.BillingUser/Infrastructure/BillingRepository.cs:673]: I think that if ExtreCreditsPromortion is null, the sum operation could be return an exception

File: Doppler.BillingUser/Infrastructure/BillingRepository.cs [hunk 2]
```diff
@@ -569,7 +569,8 @@ public async Task<int> CreateAccountingEntriesAsync(AgreementInformation agreeme
                 IdUserTypePlan = newUserTypePlan.IdUserTypePlan,
                 PlanFee = (double?)agreementInformation.Total,
                 CreditsQty = newUserTypePlan.EmailQty ?? null,
-                ExtraEmailFee = newUserTypePlan.ExtraEmailCost ?? null
+                ExtraEmailFee = newUserTypePlan.ExtraEmailCost ?? null,
+                ExtraCreditsPromotion = promotion.ExtraCredits,
```
Comment (nreal) [inline Doppler.BillingUser/Infrastructure/BillingRepository.cs:573]: The last ` ,` is not necessary

File: Doppler.BillingUser/Infrastructure/BillingRepository.cs (hunk 1 above)
Comment (jhoffman-ms) [reply Doppler.BillingUser/Infrastructure/BillingRepository.cs]: Done

File: Doppler.BillingUser/Infrastructure/BillingRepository.cs (hunk 2 above)
Comment (jhoffman-ms) [reply Doppler.BillingUser/Infrastructure/BillingRepository.cs]: Done

**Pull request merged**